### ISSUE TITLE
BIG: Use more fine-grained storage in js-sdk (i.e. not one big blob created by SyncAccumulator)

### ISSUE THREAD

**Issue by bwindels** [vector-im/riot-web#13243]

**Comment by t3chguy** [comment]
> Did you manually add it to the triage backlog, and if so, what were you hoping would happen?

Yes, all pre-triage era issues need to gradually be triaged via the backlog, I was hoping it'd be triaged like most other task-level issues, this one relating also to performance & probably data consistency.
